src/Discord.Net.Commands/Module.cs

```diff
@@ -63,9 +61,15 @@ private void SearchClass(TypeInfo parentType, object instance, List<Command> com
                 {
                     string nextGroupPrefix;
                     if (groupAttrib.Prefix != null)
+                    {
+                        if (groupPrefix != "")
```

Comment:
This is inside the type loop. Wont this add multiple spaces if there are multiple commands in a class?